pkg/ebpf/c/tracee.bpf.c

```diff
@@ -5274,6 +5276,14 @@ int BPF_KPROBE(trace_security_socket_accept)
         save_args(&args, SOCKET_ACCEPT);
     }
 
+    // save sockets for "security_socket_accept event"
+    if (should_submit(SECURITY_SOCKET_ACCEPT, &p)) {
+        args_t args = {};
```

Comment:
No need for these lines below. This should be like in the other places - if !should_submit() return 0;